Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static float CalculateBMI(float height, float weight)
+        {
+            // Calculate BMI
+            // Formula: weight (kg) / [height (m)]**2
+
+            //float heightM = height / 100;  // calculate height in meters
+
+            float BMI = weight / (height * height);
+
+            return BMI;
+        }
+
+        public static string ReadInputString(string message)
+        {
+            Console.Write(message);
+            string value = Console.ReadLine();
+            return value;
+        }
+
+        public static float ReadInputFloat(string message)
+        {
+            Console.Write(message);
+            bool _ = float.TryParse(Console.ReadLine(), out float value);
+            return value;
+        }
+
+        public static int ReadInputInt(string message)
+        {
+            Console.Write(message);
+            bool _ = int.TryParse(Console.ReadLine(), out int value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable '_'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885831)